docs/modules/ROOT/pages/runbooks/vshn-mariadb.adoc

```diff
@@ -0,0 +1,118 @@
+= MariaDB by VSHN
+
+
+[[uptime]]
+== Uptime
+
+[IMPORTANT]
+We don't yet have a lot of operational experience with this service.
+If you received this alert, please add any insights you gained to improve this runbook.
+
+=== icon:glasses[] Overview
+
+The SLI measures the uptime of each MariaDB by VSHN instance.
+This SLI is measured by a prober that executes a SQL query every second.
+
+If this SLI results in an alert, it means that a significant number of SQL queries failed and that we risk missing the SLO.
+
+There are two types of alerts that fire if we expect to miss the configured objective.
+
+* A ticket alert means that the error rate is slightly higher than the objective.
+If we don't intervene at some point after receiving this alert, we can expect to miss the objective.
+However, no immediate, urgent action is necessary.
+A ticket alert should have a label `severity: warning`.
+* A page alert means that the error rate is significantly higher than the objective.
+Immediate action is necessary to not miss the objective.
+
+=== icon:bug[] Steps for debugging
+
+Failed probes can have a multitude of reasons, but in general there are two different kinds of issue clases.
+Either the instance itself is failing or provisioning or updating the instance failed.
+
+In any case, you should first figure out where the effected instance runs.
+The alert will provide you with three labels: `cluster_id`, `namespace`, and `name`.
+
+Connect to the Kubernetes cluster with the provided `cluster_id` and get the effected claim.
+
+[source,shell]
+----
+export NAMESPACE={{ namespace }}
+export NAME={{ name }}
+
+export COMPOSITE=$(kubectl -n $NAMESPACE get vshnmariadb $NAME -o jsonpath="{.spec.resourceRef.name}")
+kubectl -n $NAMESPACE get vshnmariadb $NAME
+----
+
+If the claim is not `SYNCED` this might indicate that there is an issue with provisioning.
+If it is synced there is most likely an issue with the instance itself, you can skip to the next subsection.
+
+==== Debugging Provisioning
+
+To figure out what went wrong with provisioning it usually helps to take a closer look at the composite.
+
+[source,shell]
+----
+kubectl --as cluster-admin describe xvshnmariadb $COMPOSITE
+----
+
+If there are sync issues there usually are events that point to the root cause of the issue.
+
+Further it can help to look at the `Object` resources that are created for this instance.
```

Comment:
Maybe also mention the `Release` object, as that is basically the core of the whole service, so it's worth checking as well :D